templates/CRM/common/navigation.js.tpl

```diff
@@ -54,30 +54,25 @@
 {/strip}{/capture}// <script> Generated {$smarty.now|date_format:'%d %b %Y %H:%M:%S'}
 {literal}
 (function($) {
-  var menuMarkup = {/literal}{$menuMarkup|@json_encode};
-{if $config->userFramework neq 'Joomla'}{literal}
-  $('body').append(menuMarkup);
-
-  $('#civicrm-menu').css({position: "fixed", top: "0px"});
+  var menuMarkup = {/literal}{$menuMarkup|@json_encode}{literal};
 
   //Track Scrolling
-  $(window).scroll(function () {
-    $('div.sticky-header').css({top: "23px", position: "fixed"});
-  });
-
-  if ($('#edit-shortcuts').length > 0) {
-    $('#civicrm-menu').css({'width': '97%'});
+  if ($('div.sticky-header').length) {
+    $(window).scroll(function () {
+      $('div.sticky-header').css({top: "23px", position: "fixed"});
+    });
```

Comment:
This looks like a Drupal hack to me which is not at all related to the menu. Doesn't have to be done in this PR but on the wishlist for removal from this file.